libraries/testbot/index.js

```diff
@@ -60,6 +33,12 @@ server.listen(process.env.port || process.env.PORT || 3978, function() {
     console.log(`\n${ server.name } listening to ${ server.url }`);
 });
 
+server.post('/api/testbot', (req, res) => {
```

Comment:
This should be `mybot` to match [dotnet implementation](https://github.com/microsoft/botbuilder-dotnet/blob/f5bb7e9fdddc4968ac1668397ad020362ecae693/tests/Microsoft.Bot.Builder.TestBot/Startup.cs#L63)

Reply:
Thx, Changes applied.